### PR TITLE
Update from CVPR

### PR DESCRIPTION
**Pull request by fel-thomas**
# CVPR Presentation update & Bug Fix

Following our CVPR presentation, I propose an update using our new assets.
c19dc8be3e25540d71a19cfe231b30dc6a2f3678 introduce our new logo
11a61e9d7453d9738ebb058d334ecf8e256f8a05 put the cvpr citation in the readme / doc
9228cfd2590fc4ff3249f43b3febdfb5d15ccf17 use the new assets from the poster for the image of the notebook
ac78cb19d1488ebb28970876e046464a8e797f63 add DS_Store files to our gitignore

03d00ac26ef6240666be4843dcc2cf93816b8c55  Also, one bad behaviour that we had is fixed: the default `target_layer` should be consistent across the library. It is now `None`, which mean that by default we don't touch to the model.

### CODE REVIEW

**Review by AntoninPoche — COMMENTED** [review]
There seems to be a duplicated file between `docs/assets/concepts.jpeg` and `docs/concepts.jpg`.

Appart from this, everything looks good to me, it will be an even more welcoming library !

**Review by lucashervier — APPROVED** [review]
LGTM! New banner and images are definitely more sympathic!

File: README.md [hunk 1]
```diff
@@ -279,6 +286,21 @@ He then received help from some members of the <a href="https://www.deel.ai/"> D
 to enhance the library namely from [Justin PLAKOO](https://github.com/justinplakoo) and
 [Lucas HERVIER](https://github.com/lucashervier).
 
+## 🗞️ Citation
+
+If you use Xplique as part of your workflow in a scientific publication, please consider citing the [Xplique official paper](https://arxiv.org/abs/2206.04394):
```
Comment (thib-s) [inline README.md:291]: duplicated line:
```suggestion
```

File: docs/index.md [hunk 2]
```diff
@@ -280,6 +287,23 @@ He then received help from some members of the <a href="https://www.deel.ai/"> D
 to enhance the library namely from [Justin PLAKOO](https://github.com/justinplakoo) and
 [Lucas HERVIER](https://github.com/lucashervier).
 
+## 🗞️ Citation
+
+If you use Xplique as part of your workflow in a scientific publication, please consider citing the [Xplique official paper](https://arxiv.org/abs/2206.04394):
```
Comment (thib-s) [inline docs/index.md:292]: same here, duplicated line:
```suggestion
```

**Review by thib-s — APPROVED** [review]
found a duplicated line in `README.md` otherwise LGTM ! 👍

File: docs/index.md (hunk 2 above)
Comment (fel-thomas) [reply docs/index.md:292]: Done, thx Thib ! ;)

**Comment by fel-thomas** [comment]
> There seems to be a duplicated file between `docs/assets/concepts.jpeg` and `docs/concepts.jpg`.
> 
> Appart from this, everything looks good to me, it will be an even more welcoming library !

Done, thx Antonin ! ;)

**Review by Agustin-Picard — APPROVED** [review]
LGTM, awesome work, as usual!

**Pull request merged**